### PR TITLE
test: add test cases

### PR DESCRIPTION
**Pull request by NicoYuan1986**
1. modify test cases about output a non-existed field. issue: #28465
2. add cases about creating collection using different language
3. add test case about inserting a string value to a pk(int64) field
4. add test case about search with expression using double quotes. issue: #28365

### CODE REVIEW

File: tests/python_client/testcases/test_insert.py [hunk 1]
```diff
@@ -1253,6 +1253,21 @@ def test_insert_ids_invalid(self):
...
+    def test_insert_string_to_int64_pk_field(self):
+        """
+        target: test insert, with using auto id is invalid, which are not int64
+        method: create collection and insert entities in it
+        expected: raise exception
+        """
+        collection_name = cf.gen_unique_str(prefix)
+        collection_w = self.init_collection_wrap(name=collection_name)
+        df = cf.gen_default_dataframe_data(1)
+        df[ct.default_int64_field_name] = ["2000000"]
```
Comment (yanliang567) [inline tests/python_client/testcases/test_insert.py:1266]: I'd suggest gen 100 entities, and update one of them to be a string.

File: tests/python_client/testcases/test_search.py [hunk 2]
```diff
@@ -3350,6 +3353,39 @@ def test_search_with_comparative_expression(self, _async):
...
+
+        # 2. create index
+        index_param = {"index_type": "FLAT", "metric_type": "COSINE", "params": {}}
+        collection_w.create_index("float_vector", index_param)
+        collection_w.load()
+
+        # 3. search with expression
+        _id = random.randint(0, default_nb)
+        string_value[_id] = string_value[_id].replace("\"", "\\\"")
+        expression = f"{default_string_field_name} == \"{string_value[_id]}\""
```
Comment (yanliang567) [inline tests/python_client/testcases/test_search.py:3379]: shall we also add more scenarios for multiple single/double quotes in the inserted string

File: tests/python_client/testcases/test_insert.py (hunk 1 above)
Comment (NicoYuan1986) [reply tests/python_client/testcases/test_insert.py:1266]: Good advice, I have already changed it.

File: tests/python_client/testcases/test_search.py (hunk 2 above)
Comment (NicoYuan1986) [reply tests/python_client/testcases/test_search.py:3380]: added.

**Pull request merged**